From 7449085e076e9db1015a3fe1b0a0e162968924bc Mon Sep 17 00:00:00 2001
From: Ulises Mendez Martinez <ulises.mendez.martinez@intel.com>
Date: Mon, 11 Apr 2016 10:48:09 -0500
Subject: [PATCH 5/5] input: touch: silead: Initialize touch_nr variable before
 use it

'touch_nr' is used uninitialized.

Change-Id: Ic0636fb10a268d66a69ffa70bf65aa5d3fd0e290
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-28277
Signed-off-by: Ulises Mendez Martinez <ulises.mendez.martinez@intel.com>
Reviewed-on: https://android.intel.com:443/493170
---
 drivers/input/touchscreen/silead.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/silead.c b/drivers/input/touchscreen/silead.c
index e430ea5..391fe7b 100644
--- a/drivers/input/touchscreen/silead.c
+++ b/drivers/input/touchscreen/silead.c
@@ -203,7 +203,7 @@ static void silead_ts_read_data(struct i2c_client *client)
 	u8 buf[SILEAD_TS_DATA_LEN];
 	char read_buf[4];
 	u8 buf_t[4] = {0};
-	int x, y, id, touch_nr, ret, i, offset;
+	int x, y, id, touch_nr = 0, ret, i, offset;
 	struct gsl_touch_info cinfo = { { 0 } };
 	ret = i2c_smbus_read_i2c_block_data(client, SILEAD_REG_DATA,
 					    SILEAD_TS_DATA_LEN, buf);
-- 
1.9.1

